package.json

```diff
@@ -1,25 +1,23 @@
 {
   "name": "coala-bears",
-  "version":"0.8.0",
+  "version": "0.10.0.dev99999999999999",
   "dependencies": {
     "alex": "~3",
     "autoprefixer": "~6",
-    "babel-eslint": "~6",
```

Comment:
you cant delete *any* packages.  they are here for a reason.  you need to generate an equivalent file.
these are part of the eslint bear,